Formula/n/nodenv.rb

```diff
@@ -23,29 +23,36 @@ class Nodenv < Formula
   depends_on "node-build"
 
   def install
-    inreplace "libexec/nodenv" do |s|
-      s.gsub! "/usr/local", HOMEBREW_PREFIX
-      s.gsub! '"${BASH_SOURCE%/*}"/../libexec', libexec
-    end
-
-    %w[--version hooks versions].each do |cmd|
-      inreplace "libexec/nodenv-#{cmd}", "${BASH_SOURCE%/*}", libexec
-    end
-
-    # Compile bash extension
-    system "src/configure"
-    system "make", "-C", "src"
+    inreplace "libexec/nodenv", "/usr/local", HOMEBREW_PREFIX
 
     if build.head?
       # Record exact git revision for `nodenv --version` output
       inreplace "libexec/nodenv---version", /^(version=.+)/,
                                            "\\1--g#{Utils.git_short_head}"
     end
 
-    prefix.install "bin", "completions", "libexec"
+    prefix.install ["bin", "libexec", "nodenv.d"]
+    bash_completion.install "completions/nodenv.bash"
+    zsh_completion.install "completions/_nodenv"
```

Comment:
The rbenv formula doesn't use the `bash_completion` helper. Instead, it installs zsh completion first, then prefix.install includes "completions" directory.

https://github.com/Homebrew/homebrew-core/blob/7dfa36e1ce5060352a1b5238ac22abc41c069704/Formula/r/rbenv.rb#L28-L29

This seems awkward to me. If `prefix.install "completions"` already installs the completions for bash, would it not also install for zsh automatically? If so, we should be able to remove the zsh_completion call? Or if we prefer to be explicit with the completion helpers, should we instead use `bash_completion` helper directly in the rbenv formula as we do here?

Reply:
Yes, it's a bit awkward. Avoiding using the `bash_completion` helper actually means that the completions aren't installed into a location that makes it easily accessible to `bash`.

That might be deliberate, perhaps to avoid breaking users of older versions of `bash`.